src/main/java/reposense/git/GitClone.java

```diff
@@ -94,6 +95,8 @@ public static void cloneShallowPartialBare(RepoConfiguration config, Path rootPa
 
     /**
      * Clones repo specified in the {@code config} and updates it with the branch info.
+     *
+     * @throws GitCloneException when an error occurs during command execution.
```

Comment:
I think this description can be more explicit. Maybe something like `GitCloneException when an error occurs while attempting to clone the repo`. Though I realise this exact line is used multiple times over the entire file so if you were to change this, might have to change the rest as well.

The wording currently used sounds it would fit a more generic `CommandRunnerProcessException` instead.

Reply:
Thanks for pointing it out. I have made the change accordingly.